patch_creation_wizard

```diff
@@ -105,15 +105,16 @@ def generate_patch_diff_files(patch_name, path_to_repo, commit):
             cwd=path_to_repo,
         )
         diff_file_name = patch_name + '_' + different_file.replace('/', '_')
-        diff_path = os.path.join('patches', diff_file_name)
+        diff_path = os.path.join('patch_files', 'patches', diff_file_name)
         with open(diff_path, 'w') as diff_handle:
             diff_handle.write(diff)
         diff_files[diff_file_name] = {
             'path_in_repo': different_file,
             'before_md5sum': None,
             'after_md5sum': None,
             'diff_md5sum': subprocess.check_output(
-                ['md5sum', os.path.join('patches', diff_file_name)],
+                ['md5sum',
+                 os.path.join('patch_files', 'patches', diff_file_name)],
```

Comment:
maybe const 'patch_files', 'patches' to avoid copy paste errors and a chance to doc them

Reply:
Moved to a constant.